container/containerd/handler.go

```diff
@@ -112,11 +116,22 @@ func newContainerdContainerHandler(
 		rootfs = "/rootfs"
 	}
 
+	aliases := []string{id, name}
+
+	if *useK8sContainerNameFormat && cntr.Labels["io.cri-containerd.kind"] == "container" {
+		restart := cntr.Labels["io.kubernetes.pod.restart"]
+		if restart == "" {
+			restart = "0"
+		}
+		containerNameConcat := "k8s_" + cntr.Labels["io.kubernetes.container.name"] + "_" + cntr.Labels["io.kubernetes.pod.name"] + "_" + cntr.Labels["io.kubernetes.pod.namespace"] + "_" + cntr.Labels["io.kubernetes.pod.uid"] + "_" + restart
```

Comment:
strings.Join looks cleaner like here https://github.com/kubernetes/kubernetes/blob/ab595f37d258b85291bb6f4f18a1649558346bc0/pkg/kubelet/dockershim/naming.go#L60-L80, but I don't think there is a difference

Reply:
Sounds good. Will change to strings.join

Reply:
+1

Reply:
Please also leave a comment where these labels are defined, i.e. https://github.com/kubernetes/kubernetes/blob/ab595f3/pkg/kubelet/types/labels.go#L24